README.md

```diff
@@ -185,6 +185,33 @@ You can get an equivalent of the Command Line Tool using the Python3 executable
 $ python3 -m pyshacl
 ```
 
+## Errors
+Under certain circumstances pySHACL can produce a `Validation Failure`. This is a formal error defined by the SHACL specification and is required to be produced as a result of specific conditions within the SHACL graph.
+If the validator produces a `Validation Failure`, the `results_graph` variable returned by the `validate()` function will be an instance of `ValidationFailure`.
+See the `message` attribute on that instance to get more information about the validation failure.
+
+Other errors the validator can generate:
+- `ShapeLoadError`: This error is thrown when a SHACL Shape in the SHACL graph is in an invalid state and cannot be loaded into the validation engine.
+- `ConstraintLoadError`: This error is thrown when a SHACL Constraint Component is in an invalid state and cannot be loaded into the validation engine.
+- `ReportableRuntimeError`: An error occurred for a different reason, and the reason should be communicated back to the user of the validator.
+- `RuntimeError`: The validator encountered a situation that caused it to throw an error, but the reason does concern the user.
```

Comment:
Double-checking this remark: "...but the reason does concern the user."  Should this be "and the reason does concern the user," "but the reason does not concern the user," or as-is?

Reply:
Thanks. It should be "does not".